move-vm-backend-common/src/gas_schedule.rs

```diff
@@ -28,122 +28,122 @@ lazy_static! {
     /// A predefined gas strategy for instruction table cost.
     pub static ref INSTRUCTION_COST_TABLE: CostTable = {
         let mut instrs = vec![
-        (MoveTo(StructDefinitionIndex::new(0)), GasCost::new(13, 1)),
+        (MoveTo(StructDefinitionIndex::new(0)), GasCost::new(1838, 1)),
         (
             MoveToGeneric(StructDefInstantiationIndex::new(0)),
-            GasCost::new(27, 1),
+            GasCost::new(1838, 1),
         ),
         (
             MoveFrom(StructDefinitionIndex::new(0)),
-            GasCost::new(459, 1),
+            GasCost::new(1286, 1),
         ),
         (
             MoveFromGeneric(StructDefInstantiationIndex::new(0)),
-            GasCost::new(13, 1),
+            GasCost::new(1286, 1),
         ),
-        (BrTrue(0), GasCost::new(1, 1)),
-        (WriteRef, GasCost::new(1, 1)),
-        (Mul, GasCost::new(1, 1)),
-        (MoveLoc(0), GasCost::new(1, 1)),
-        (And, GasCost::new(1, 1)),
-        (Pop, GasCost::new(1, 1)),
-        (BitAnd, GasCost::new(2, 1)),
-        (ReadRef, GasCost::new(1, 1)),
-        (Sub, GasCost::new(1, 1)),
-        (MutBorrowField(FieldHandleIndex::new(0)), GasCost::new(1, 1)),
+        (BrTrue(0), GasCost::new(441, 1)),
+        (WriteRef, GasCost::new(735, 1)),
```

Comment:
We could nullify the second parameter since we know it's only summed up with the first argument - and explain this with a quick comment. We could even mention that these values were taken from `aptos-core` here in the code as well.
```suggestion
        (BrTrue(0), GasCost::new(441, 0)),
        (WriteRef, GasCost::new(735, 0)),
```
Since we want the `GasCost::total()` to return 441, not 442 in the case of `BrTrue`, e.g.

Reply:
Done :+1: